src/openzaak/templates/admin/submit_line.html

```diff
@@ -6,5 +6,9 @@
         <input type="submit" value="{% trans 'Publish' %}" name="_publish"/>
     {% endif %}
 
+    {% if original.domein %}
+        <input type="submit" value="{% trans 'Export' %}" name="_export"/>
```

Comment:
This becomes messy now with all the actions and I definitly don't like the if-statement nor the attribute checking. But you can leave it for now and we'll see how to make it more elegant afterwards. I made #268 for this.

Reply:
Since #268 was merged, this branch will have merge conflicts with this change.

You can copy (in master branch) `templates/admin/catalogi/change_form_zaaktype.html` to `change_form_catalog.html` and use the same template logic to add this button without the if-statement.

Then use this template by specifying `change_form_template = "admin/catalogi/change_form_catalog.html"` in the 
`CatalogusAdmin` class.